### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin @sdroege

### CODE REVIEW

File: src/pixbuf.rs [hunk 1]
```diff
@@ -106,8 +106,7 @@ impl Pixbuf {
         }
     }
 
-    pub fn new_from_stream_async<'a, P: IsA<gio::InputStream>, Q: Into<Option<&'a gio::Cancellable>>, R: FnOnce(Result<Pixbuf, Error>) + Send + 'static>(stream: &P, cancellable: Q, callback: R) {
-        let cancellable = cancellable.into();
+    pub fn new_from_stream_async<'a, P: IsA<gio::InputStream>, R: FnOnce(Result<Pixbuf, Error>) + Send + 'static>(stream: &P, cancellable: Option<&gio::Cancellable>, callback: R) {
```
Comment (sdroege) [inline src/pixbuf.rs]: This should be `Option<IsA<gio::Cancellable>`

**Comment by GuillaumeGomez** [comment]
Updated.

File: src/pixbuf.rs (hunk 1 above)
Comment (sdroege) [reply src/pixbuf.rs]: Obviously everywhere not just in this single function :)

Comment (GuillaumeGomez) [reply src/pixbuf.rs]: I had hopes... :-.

**Pull request merged**